docs/excel/add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,398 @@
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const range = sheet.getRange("A1");
+    const attributionObject: Excel.CellValueAttributionAttributes = {
+      licenseAddress: "https://en.wikipedia.org/wiki/Wikipedia:Wikipedia_is_free_content",
+      licenseText: "Free usage information",
+      sourceAddress: "https://en.wikipedia.org/wiki/Mars",
+      sourceText: "Wikipedia"
+    };
+
+    range.valuesAsJson = [
+      [
+        {
+          type: Excel.CellValueType.double,
+          basicType: Excel.RangeValueType.double,
+          basicValue: 6779, //kilometers (radius)
+          properties: {
+            Name: {
+              type: Excel.CellValueType.string,
+              basicType: Excel.RangeValueType.string,
+              basicValue: "Mars",
+              propertyMetadata: {
+                sublabel: "Planetary Body",
+                attribution: [attributionObject]
+              }
+            }
+          }
+        }
+      ]
+    ];
+    await context.sync();
+  });
+}
+
+```
+
+The following image shows how the attribution is displayed in the data type card for the user.
+
+:::image type="content" source="../images/data-type-basic-card-attribution.png" alt-text="Data type card showing attribution for Wikipedia.":::
+
+## Provider information
+
+You can add information about your add-in, or service, that is the source for the information in the data type card. Use **Excel.CellValueProviderAttributes** to add your provider information. The following code sample shows how to add provider information for Contoso generic search as the source of search data for the cell value.
+
+```javascript
+async function createSearchEntry() {
+  await Excel.run(async (context) => {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const range = sheet.getRange("A1");
+
+    range.valuesAsJson = [
+      [
+        {
+          type: Excel.CellValueType.string,
+          basicType: Excel.RangeValueType.string,
+          basicValue: "cell function - Microsoft support",
+          properties: {
+            "Search Keywords": {
+              type: Excel.CellValueType.string,
+              basicType: Excel.RangeValueType.string,
+              basicValue: "Cell Values"
+            }
+          },
+          provider: {
+            description: "Contoso generic search",
+            //Ignacio javier igjav, Public domain, via Wikimedia Commons
+            logoSourceAddress: "https://upload.wikimedia.org/wikipedia/commons/f/f9/Lupa.png",
+            logoTargetAddress: "https://contoso.com"
+          }
+        }
+      ]
+    ];
+
+    await context.sync();
+  });
+}
+```
+
+The following image shows how the provider information appears as the logo in the data type card for the user.
+
+:::image type="content" source="../images/data-type-basic-provider-information.png" alt-text="Data type card showing the search logo for Contoso generic search.":::
+
+## Compatibility
```

Comment:
Should we put this section on the [Overview](https://learn.microsoft.com/en-us/office/dev/add-ins/excel/excel-data-types-overview) article? It seems relevant to all the data types